arangod/Aql/grammar.cpp

```diff
@@ -1,9 +1,9 @@
 /* clang-format off */
-/* A Bison parser, made by GNU Bison 3.8.2.  */
+/* A Bison parser, made by GNU Bison 3.5.1.  */
```

Comment:
Is it intentional grammar regeneration? I was sure we don't need to change grammar in this PR.